src/app/child-dev-project/notes/note-detail/note-detail.component.html

```diff
@@ -0,0 +1,140 @@
+      </div>
+      <div fxFlex>
+        <mat-form-field class='input-medium'>
+          <!-- TODO: should be multi-select or auto-complete field of user names? -->
+          <input matInput placeholder="Team Member" name="teammember" type="text" [(ngModel)]="entity.author">
+        </mat-form-field>
+      </div>
+      </div>
+
+    <div fxLayout='row wrap' fxLayoutGap='20px'>
+      <div fxFlex>
+        <app-child-select [valueAsIds]='entity.getChildIDs()'
+                          (newIdAdded)="entity.addChild($event);recordForm.form.markAsDirty()"
+                          (idRemoved)="entity.removeChild($event);recordForm.form.markAsDirty()"
+                          class='input-medium'>
+        </app-child-select>
+      </div>
+    </div>
+    </div>
+
+
+
+  <!-- Primary information of Note -->
+  <div class='mat-elevation-z2 primary-section-elevated form-section-primary'>
+    <div fxLayout='row wrap' fxLayoutGap='20px'>
+      <div fxFlex>
+        <mat-form-field class='input-medium'>
+          <input matInput placeholder="Topic / Summary" name="subject" type="text" [(ngModel)]="entity.subject">
+        </mat-form-field>
+      </div>
+    </div>
+
+    <div fxLayout='row wrap' fxLayoutGap='20px'>
+      <mat-form-field class='input-max'>
+      <textarea matInput placeholder="Notes" name="notes" [(ngModel)]="entity.text"
+                cdkTextareaAutosize cdkAutosizeMinRows="4"></textarea>
+      </mat-form-field>
+    </div>
+  </div>
+
+    <!--Small screen: display the notes in tabs-->
+    <div *ngIf="entity.isMeeting() && smallScreen" fxLayout="row wrap" fxLayoutGap="20px">
+
+      <mat-tab-group mat-align-tabs="center" fxFlex>
+        <mat-tab label="Present" class="padding-container" >
+          <app-child-presence-list [note]="entity" [recordForm]="recordForm" [present]="true" fxFlex></app-child-presence-list>
+        </mat-tab>
+        <mat-tab label="Absent" class="padding-container" >
+          <app-child-presence-list [note]="entity" [recordForm]="recordForm" [present]="false" fxFlex></app-child-presence-list>
+        </mat-tab>
+      </mat-tab-group>
+
+    </div>
+
+    <!--Big screen: display the notes as two lists-->
+    <div *ngIf="entity.isMeeting() && !smallScreen" fxLayout="row wrap" fxLayoutGap="20px">
+      <app-child-presence-list [note]="entity"
+                               [recordForm]="recordForm"
+                               [present]="true"
+                               [label]="'Present'"
+                               class="padding-bottom"
+                               fxFlex>
+      </app-child-presence-list>
+      <app-child-presence-list [note]="entity"
+                               [recordForm]="recordForm"
+                               [present]="false"
+                               [label]="'Absent'"
+                               class="padding-bottom"
+                               fxFlex>
+      </app-child-presence-list>
+    </div>
```

Comment:
Let's extract this into a separate component as well so the responsive version can be easily re-used (and the template here will be shorter and more manageable). Possibly such a new component could also include the `app-child-select` component above and be somewhat of an alternative to the simple `app-child-select`